yaml/csi-driver/edge/hpe-csi-k8s-1.20.yaml

```diff
@@ -176,6 +176,9 @@ spec:
             - "--flavor=kubernetes"
             - "--pod-monitor"
             - "--pod-monitor-interval=30"
+            {{- if .Values.cspClientTimeout }}
+            - "--csp-client-timeout={{ .Values.cspClientTimeout }}"
```

Comment:
template functions do not work on the yaml files.

Reply:
@raunakkumar Done.